ipamd/ipamd.go

```diff
@@ -560,6 +567,11 @@ func (c *IPAMContext) increaseIPPool() {
 		return
 	}
 
+	if c.isTerminating() {
+		log.Info("AWS CNI is terminating, will not try to attach any new IPs or ENIs right now")
```

Comment:
on line 566 above, we use Debug level, not Info, for what is essentially the same message (skipping increasing IP pool because of X). Probably good to be consistent and use Debug here.

Reply:
True, will update.